### ISSUE TITLE
Suggestion - Make upgrading basic repair kits to advanced repair kits cheaper(and maybe easier to see?).

### ISSUE THREAD

**Issue by MAGiCxPOTATOE** [InZidiuZ/op-framework-issue-tracker#2891]
### Assurance

I have read the `README.md`

### Summary

I'm not exactly sure when it was changed but as far as I know as well as Mosley's staff, we remember doing the upgrade to basic repair kits took, 1 fuel bottle, 1 hammer, 1 wrench and 1 basic which cost about $438(given the you purchase the basic for $175 from another player). Now it takes an extra hammer which costs $545(again with basic bought for $175). Most mechanics sell them for $550-$600. To craft a advanced from scratch using only material bought from the computer costs $515. This is cheaper than upgrading even after sourcing and interacting with a player to purchase a basic and then getting all the other parts and crafting it!
The cost for making an advanced from materials bought from people alone is $398.
So I propose that only 1 hammer be needed for the craft.

For the making easier to see the upgrade could be within the workbench unless it is something that you want mechs to find out in RP but even when I chat to other mechanics at different shops and they find out you can do the upgrade they are not sure if they know where to make it or even where it could be. Cause I know there are some awkward areas like in Mosley's where it used to be in the old front area that had to be access through the wall.

### Reason

This is for a few reason that I will try my best to bullet point.
1. Cost balancing: You can see that using material bought from people it is much cheaper from the $398 price point to craft. So for upgrading the basics I guess you could say you could buy hammers and wrenches for cheaper from people but who has even got a wrench, hammer or any melee weapon and though of selling it on, even a decent amount of mechanics don't know about upgrading basics.
2. Interactions: I can see why the craft from scratch is about $110 cheaper because of the fact that you have to source and talk to people about items/prices. So I think the upgrade price should be similar because for upgrading to be viable you have to find and talk to people about getting the crafting parts.
3. Time Saving/ Convenience: I prefer to have my stuff prepped and ready for mech work like having materials always filled by buying them from people and the same is for advanced repair kits. No one enjoys making or waiting for mechanics to make them the advanced repair kits.

Overall I think that this change would help encourage interactions with people because right now I just make advanced in bulk the easiest/cheapest way which is by buying the materials from the computer because no one wants to spend purchased materials on advanced cause it really messes with the balance of materials you need down the line which then ends up shafting materials gatherers/sellers because mechs need to then pick and choose certain mats to keep even materials. 

I have spoke to a few mechs about this and they seem to think it makes sense, although as far as I could tell I am like the only person that I know of that actually tries to use the upgrade feature again which is why I think it should be made easier to see in the workbench).

TL:DR: Current Upgrade expensive/time consuming, buying from computer easier. Make upgrade cheaper and more well known about so people can use it. Thus, making it still time consuming, but more cost worthy

**Comment by MetaHatter** [comment]
Mechanics can make from mats.  Anyone can make from repair kits.  The price increase was done to make purchasing from mechanics, that make them w/mats, more appealing.

**Comment by MetaHatter** [comment]
IMHO Unlikely as this is the means for people to acquire kits when there are no mechanics online.  That being said, if you spend the mats on repairs or on kits, the overall costs should balance out.  Mat requirements are there to incentivize people to harvest and sell them. Worst case, you should still be able to make **some** money buying from the computer.

It's one of those that needed to be discovered, but the info on system mechanics always finds a way into the population.  So now it's more about asking the right people.  The mysteries never last too long.

**Comment by MAGiCxPOTATOE** [comment]
Okay okay I get ya, I get that stuff finds a way into the population but 90% of the time it seems people see changelogs and start religiously hunting down new things they see, although maybe I am just out of the loop!💀
thanks for your time tho I appreciate the responses. I shall now close this
